graphistry/feature_utils.py

```diff
@@ -91,6 +101,25 @@ def lazy_import_has_min_dependancy():
     except ModuleNotFoundError as e:
         return False, e
 
+def lazy_import_has_cu_cat_dependancy():
```

Comment:
i think the callers are still calling `lazy_import_has_cuml_dependency()` instead of the new proper name here